Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -364,12 +392,18 @@ PatchAppleXcpmExtraMsrs (
   //
   // Now patch writes to MSR_MISC_PWR_MGMT
   //
-  Status = PatcherApplyGenericPatch (Patcher, &mMiscPwrMgmtPatch);
+  Status = PatcherApplyGenericPatch (Patcher, &mMiscPwrMgmtRelPatch);
+  if (RETURN_ERROR (Status)) {
+    DEBUG ((DEBUG_WARN, "OCAK: Failed to patch writes to MSR_MISC_PWR_MGMT - %r, trying dbg\n", Status));
+    Status = PatcherApplyGenericPatch (Patcher, &mMiscPwrMgmtDbgPatch);
+  }
+
   if (!RETURN_ERROR (Status)) {
     ++ Replacements;
```

Comment:
Should be `++Replacements;` without space probably, and add a `DEBUG_INFO` log here, so that we have in the log that the patch has happened. Thanks!